lumicks/pylake/kymotracker/detail/trace_line_2d.py

```diff
@@ -97,15 +97,56 @@ def _traverse_line_direction(indices, masked_derivative, positions, normals, con
     return nodes
 
 
+class KymoLine:
+    """A line on a kymograph"""
+    __slots__ = ['time', 'coordinate']  # Fixed attributes
```

Comment:
No need to comment on language features. Anyone who doesn't know it will need to google it anyway. `Fixed attributes` will not exactly explain `__slots__`.

Reply:
True.